host_core/lib/host_core/actors/actor_rpc_supervisor.ex

```diff
@@ -0,0 +1,62 @@
+defmodule HostCore.Actors.ActorRpcSupervisor do
+  require Logger
+  use Supervisor
+
+  def start_link(state) do
+    Supervisor.start_link(__MODULE__, state, name: __MODULE__)
+  end
+
+  def init(_opts) do
+    Process.flag(:trap_exit, true)
+    Supervisor.init([], strategy: :one_for_one)
+  end
+
+  defp via_tuple(pk) do
+    {:via, Registry, {Registry.ActorRpcSubscribers, pk, []}}
+  end
+
+  def stop_rpc_subscriber(public_key) do
+    if !Application.get_env(:host_core, :retain_rpc_subscriptions, false) do
+      Logger.debug("Terminating RPC subscriber for actor #{public_key}")
+      Supervisor.terminate_child(__MODULE__, via_tuple(public_key))
+    end
+  end
+
+  def start_or_reuse_consumer_supervisor(claims) do
+    prefix = HostCore.Host.lattice_prefix()
+    topic = "wasmbus.rpc.#{prefix}.#{claims.public_key}"
+
+    cs_settings = %{
+      connection_name: :lattice_nats,
+      module: HostCore.Actors.ActorRpcServer,
+      subscription_topics: [
+        %{topic: topic, queue_group: topic}
+      ]
+    }
```

Comment:
Interesting, so instead of each instance subscribing to a queue group now it's only one per-host, so the queue groups will now round robin between hosts (effectively)

Reply:
yes